config/webpack/environment.js

```diff
@@ -1,9 +1,10 @@
 const { environment } = require('@rails/webpacker')
-
+const { VueLoaderPlugin } = require('vue-loader')
 const vue = require('./loaders/vue')
-environment.loaders.append('vue', vue)
 
-// resolve-url-loader must be used before sass-loader
+environment.plugins.prepend('VueLoaderPlugin', new VueLoaderPlugin())
+environment.loaders.prepend('vue', vue)
+
 environment.loaders.get('sass').use.splice(-1, 0, {
   loader: 'resolve-url-loader'
 });
```

Comment:
You could probably replace these with prepend or append

Reply:
I probably should, but I don't actually use the resolve-url-loader currently, so I can probably get away with removing this entirely instead. I tried replacing this with prepend but it didn't quite work as expected.

For now I'll leave it in.